### PR TITLE
Tagging the GL gang

### PR DESCRIPTION
**Pull request by SimonDanisch**

### CODE REVIEW

File: GLAbstraction/versions/0.2.0/requires [hunk 1]
```diff
@@ -0,0 +1,10 @@
+julia 0.4
+Quaternions
+FixedSizeArrays
+GeometryTypes
+ModernGL
+Reactive
+FixedPointNumbers
+ColorTypes
+Compat 0.7.15
```
Comment (tkelman) [inline GLAbstraction/versions/0.2.0/requires]: this needs to be 0.8.0 for unsafe_string https://github.com/JuliaLang/Compat.jl/commit/d22506770d4b52bd2409fc70fa879e707b573949

**Comment by SimonDanisch** [comment]
Thanks tony! 
Not sure if github is trolling, or If I somehow managed push the same commits 2 times.
The diff looks fine though!

**Comment by SimonDanisch** [comment]
Hah, it was trolling I guess ;)

**Comment by tkelman** [comment]
sometimes you have to refresh if you do a rebase / force push

**Comment by SimonDanisch** [comment]
I did ;) But it took a full five minutes! Thanks!

File: GLVisualize/versions/0.1.0/requires [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+FixedPointNumbers
+FileIO
+Packing
+SignedDistanceFields
+Images
+FreeType
+MeshIO
+Compose
+Iterators
+UnicodeFun
```
Comment (tkelman) [inline GLVisualize/versions/0.1.0/requires]: ColorVectorSpace is missing here

Comment (SimonDanisch) [inline GLVisualize/versions/0.1.0/requires]: have added it ;)

Comment (tkelman) [inline GLVisualize/versions/0.1.0/requires]: also needed in the metadata version

Comment (SimonDanisch) [inline GLVisualize/versions/0.1.0/requires]: Sure! I'll get a new version ready anyways ;) Just want to make sure that I don't tag two times in half a day!

**Pull request closed without merging**